### ISSUE TITLE
Add a page on citing OpenMM?

### ISSUE THREAD

**Issue by jchodera** [openmm/openmm-org#32]
What do you think about adding a page to make it easy to cite OpenMM, like [MDTraj does](http://mdtraj.org/1.9.3/)?
![image](https://user-images.githubusercontent.com/3656088/64926726-e2cf9300-d7ce-11e9-8a5e-977136cf2edb.png)

**Comment by peastman** [comment]
You mean like https://simtk.org/plugins/publications/index.php?type=group&id=161&pluginname=publications?

**Comment by jchodera** [comment]
Yes! That's hard to find from openmm.org. Maybe we can link to that, or move it over to openm.org?

**Comment by peastman** [comment]
Sure, let's add a link.
